RxAlamofire/RxAlamofire-iOS-Example/MasterViewController.swift

```diff
@@ -24,45 +39,36 @@ class MasterViewController: UIViewController, UITextFieldDelegate {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        
-    }
-
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-    }
-
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
     }
     
     // MARK: - UI Actions
     
     @IBAction func convertPressed(_ sender: UIButton) {
         fromTextField.resignFirstResponder()
         
-        let formatter = NumberFormatter()
```

Comment:
hmm, why did we remove the number formatter?

Reply:
Hi, @sunshinejr , thanks for the review!

I removed it because the EUR number above is not formatted either, and we already have a label("USD") to indicate what the number below is, I think this is enough to clearly express what we are showing.

These two numbers should either be formatted together or not, I can bring the format back if you want! 😉

![fadskjfha](https://user-images.githubusercontent.com/18638838/66793644-ecb0f700-ef30-11e9-8c20-25d57b7af813.png)

Reply:
ah yeah gotcha, although rounding it to 2 places would be great, I agree it's not that important as it's just an example to show the API usage